README.md

```diff
@@ -15,66 +15,71 @@ characteristics:
 Public loadbalancer example:
 
 ```hcl
-variable "resource_group_name" {
-  default = "my-terraform-lb"
+provider "azurerm" {
+  features {}
 }
 
-variable "location" {
-  default = "eastus"
+resource "azurerm_resource_group" "example" {
+  name     = "example-lb"
+  location = "West Europe"
 }
 
 module "mylb" {
   source              = "Azure/loadbalancer/azurerm"
-  resource_group_name = "${var.resource_group_name}"
-  location            = "${var.location}"
+  resource_group_name = azurerm_resource_group.example.name
   prefix              = "terraform-test"
 
-  "remote_port" {
+  remote_port = {
     ssh = ["Tcp", "22"]
   }
 
-  "lb_port" {
+  lb_port = {
     http = ["80", "Tcp", "80"]
   }
 }
 
-module "network" {
-  source              = "Azure/network/azurerm"
-  location            = "${var.location}"
-  resource_group_name = "${var.resource_group_name}"
-}
 ```
 
 Private loadbalancer example:
 
 ```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-lb"
+  location = "West Europe"
+}
+
 module "mylb" {
   source                                 = "Azure/loadbalancer/azurerm"
-  location                               = "westus"
+  resource_group_name                    = azurerm_resource_group.example.name
   type                                   = "private"
-  frontend_subnet_id                     = "${module.network.vnet_subnets[0]}"
+  frontend_subnet_id                     = module.network.vnet_subnets[0]
   frontend_private_ip_address_allocation = "Static"
   frontend_private_ip_address            = "10.0.1.6"
+  lb_sku                                 = "Standard"
 
-  "remote_port" {
+  remote_port = {
     ssh = ["Tcp", "22"]
   }
 
-  "lb_port" {
-    http  = ["80", "Tcp", "80"]
-    https = ["443", "Tcp", "443"]
+  lb_port = {
+    http   = ["80", "Tcp", "80"]
+    https  = ["443", "Tcp", "443"]
+    https2 = ["1443", "Tcp", "1443", "/", "Https"]
```

Comment:
In variables.tf, the definition of "lb_port" is "...[frontend_port, protocol, backend_port]...", i.e. there are totally 3 parameters, while here the "https2" has 5 parameters. Is it a mismatch or any description needs to be changed in the variables.tf?

Reply:
I have changed the corresponding description in variables.tf